### PR TITLE
Add missing boolean operations

### PR DESCRIPTION
**Pull request by dnahodil**
Adds `nand`, `nor`, `xor` and `xnor` macros.

Ref #446

### CODE REVIEW

File: addon/xor.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import and from './and';
+import not from './not';
+import or from './or';
+
+export default function() {
+  return and(
+    or(...arguments),
+    not(and(...arguments))
```
Comment (kellyselden) [inline addon/xor.js]: nand here?

Comment (dnahodil) [reply addon/xor.js]: Oh yep. Derp.

**Comment by kellyselden** [comment]
Looks great. Thanks again!

**Comment by dnahodil** [comment]
Cool.

Cheers,

**Pull request merged**